lib/src/screens/new_wallet/new_wallet_type_page.dart

```diff
@@ -13,23 +13,22 @@ class NewWalletTypePage extends BasePage {
 
   final void Function(BuildContext, WalletType) onTypeSelected;
   final walletTypeImage = Image.asset('assets/images/wallet_type.png');
-  final walletTypeLightImage =
-      Image.asset('assets/images/wallet_type_light.png');
+  final walletTypeLightImage = Image.asset('assets/images/wallet_type_light.png');
 
   @override
   String get title => S.current.wallet_list_restore_wallet;
 
+  @override
+  Widget trailing(BuildContext context) => SizedBox.shrink();
+
```

Comment:
Remove this please, I already handled the case of null trailing in the `nav_bar.dart`